installer/pkg/components/server/configmap.go

```diff
@@ -42,8 +49,8 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 			DefaultFeatureFlags: []NamedWorkspaceFeatureFlag{},
 		},
 		Session: Session{
-			MaxAgeMs: 259200000,
-			Secret:   "Important!Really-Change-This-Key!", // todo(sje): how best to do this?
+			MaxAgeMs: int32((time.Hour * 24 * 3).Milliseconds()),
+			Secret:   cookieSecret,
```

Comment:
Yes, it would invalidate any sessions. The question is, which option do we want to pursue on the original issue?

Reply:
@csweichel I don't see this as being optional - it's a required value

Reply:
if the user configures a secret, we use that. Otherwise we use a fixed string.

Reply:
I'm tempted to create an optional `gitpod-settings` secret in the config because I can see that there's going to be quite a few little optional settings we'll want to set in the fullness of time. And then have this look for a `server_cookiesecret` value. WDYT @csweichel?

Reply:
In discussion with @gitpod-io/engineering-meta we found that we might want to add an secret which gets mounted to server which meta could use to ship all sorts of config specific to their needs.

That's a fine line however, because
- we'd be externalising config surface running the risk again for config drift in server
- it's unclear how we'd bring such values back if we find that users need to set them.

Reply:
@MrSimonEmms considering that others will want to customise this string themselves, let's go with the following behaviour:

1. by default we use a hardcoded string much like today
2. we introduce an optional secret, which when provided, sets the session secret